hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchIntegrationTest.kt

```diff
@@ -650,8 +652,205 @@ class AttributeSearchIntegrationTest : AbstractSearchDataIntegrationTest() {
 
       webTestClient.attributeSearch(request).expectPrisoners("B1234BB", "C1234CC")
     }
+
+    @Test
+    fun `AND with 2 integers`() {
```

Comment:
Added a few tests for joining integer matchers which were missed from previous PR